From 66df4b4ecfdc086d42b2d63e8cfa4b2cdca2cdfd Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Tue, 29 Apr 2025 14:28:24 +0300
Subject: [PATCH] [SW-217069] make: hls3/hlb325: avoid rsync layers

Avoid using rsynced yocto layers for openbmc, which don't properly track
the changes performed in them, instead, the original ones are used.

Change-Id: Ib2568c980f78a01581ea3b56ea6726ffb0694a68
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/scripts/make/hls3-common/Makefile.hls3-common b/scripts/make/hls3-common/Makefile.hls3-common
index 8d0387c..941c232 100644
--- a/scripts/make/hls3-common/Makefile.hls3-common
+++ b/scripts/make/hls3-common/Makefile.hls3-common
@@ -16,7 +16,9 @@
 
 define $(U_HL_DEV)_ENV
 	mkdir -p $($(U_HL_DEV)_WORK_DIR); \
-	source setup $(HL_DEV) $($(U_HL_DEV)_WORK_DIR);
+	source setup $(HL_DEV) $($(U_HL_DEV)_WORK_DIR); \
+	sed -i s#buildroot-external/output/$(HL_DEV)/build/$(HL_DEV)-bmc-custom/meta#openbmc/meta#g \
+		$($(U_HL_DEV)_DEPLOY_DIR)/build/$(HL_DEV)/conf/bblayers.conf;
 endef
 export $(U_HL_DEV)_ENV
 
